composer.json

```diff
@@ -83,7 +84,8 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "component-dir": "web/libraries"
```

Comment:
What installs component-installer? With the `type:` installer definition, I don't think it's needed. But I could be wrong!

Reply:
[RobLoach/component-installer](https://github.com/RobLoach/component-installer) was getting installed since it was a dependency of one of the components libs (jquery_ns_autogrow or chosen or something). I believe that dependency has since been removed, but I'm okay leaving this in here, as if something else ever requires it, `component-installer` overrides the [composer/installers](https://github.com/composer/installers) behaviour.